packages/ai-openai/src/common/openai-language-models-manager.ts

```diff
@@ -54,6 +54,10 @@ export interface OpenAiModelDescription {
      * Flag to configure whether the OpenAPI model supports structured output. Default is `true`.
      */
     supportsStructuredOutput: boolean;
+    /**
+     * Maximum number of retry attempts when a request fails. Default is 3.
+     */
+    maxRetries?: number;
```

Comment:
You always set this number anyways, so I would just not model it as optional. The same consideration is for all other places where this was declared optionally.